src/main/app/claims/models/claim.ts

```diff
@@ -81,6 +81,7 @@ export class Claim {
   paperResponse: YesNoOption
   claimDocuments?: ClaimDocument[]
   proceedOfflineReason: string
+  handedOffToCcbcAt: Moment
```

Comment:
Is this needed. I don't see it used anywhere?

Reply:
I initially thought date might be needed in history page, looks like date is not needed in that page

Reply:
This should be removed then

Reply:
done